src/platform/plugins/shared/chart_expressions/expression_metric/public/components/metric_vis.test.tsx

```diff
@@ -1508,4 +1508,51 @@ describe('MetricVisComponent', function () {
       expect(settingsComponent.prop('ariaUseDefaultSummary')).toEqual(true);
     });
   });
+
+  describe('ES|QL metric chart', () => {
+    const dataFromESQL: Datatable = {
+      type: 'datatable',
+      columns: [
+        {
+          id: 'a',
+          name: 'alpha',
+          meta: {
+            esType: 'keyword',
+            type: 'string',
+            sourceParams: {
+              indexPattern: 'index',
+            },
+          },
+        },
+      ],
+      rows: [{ a: '12h50m30s' }],
+      meta: {
+        type: 'es_ql',
+      },
+    };
+
+    const config: Props['config'] = {
+      dimensions: {
+        metric: 'a',
+      },
+      metric: {
+        color: '#FFFFFF',
+        iconAlign: 'left',
+        maxCols: 3,
+        titlesTextAlign: 'left',
+        valueFontSize: 'default',
+        valuesTextAlign: 'right',
+      },
+    };
```

Comment:
nit: this looks like a test specific configuration rather than a generic thing for the ES|QL scenario. Can you move them into the test itself rather than here?